### ISSUE TITLE
Change lightcurve.to_pandas() so that 'time' isn't the index by default

### ISSUE THREAD

**Issue by ojhall94** [KeplerGO/lightkurve#260]
#### Problem description
When using lightcurve.to_pandas(), it sets the values of 'time' as the index values of the newly formed pandas array. This is information is duplicate with the data in the 'time' column, and means that data manipulation using indices as you would with a numpy array is no longer possible, i.e.

```
lightcurve.to_pandas()[0:50]
```
does not work. In my experience, manipulating pandas this way is more intuitive.

#### Example
```python
from lightkurve import KeplerTargetPixelFile
tpf = KeplerTargetPixelFile.from_archive("Kepler-10", quarter=3)
df = lc.to_pandas()
df[0:50]        #Attempt to print the first 50 rows
```

#### Goal/Expected behavior
An ideal change would be to have no custom index as a default, and allow the user to pass a string, i.e. 'time' to set time (or another column) as the index column as well.

**Comment by barentsen** [comment]
Thanks for reporting this experience Oliver! My reasoning behind having `time` as index is as follows:
 
the `pandas` way to select rows by index number is to use the `iloc` attribute, e.g. `df.iloc[0:50]` prints the first 50 rows in the example above.  The index, instead, tends to be a more informative field, such as time.  This allows a user to select sub-sets more easily by time, and it allow e.g. the `df.plot()` method to show a more meaningful piece of information (time) on the X axis by default.

In summary: I do think having `time` as the index is the "pandas thing to do".

**Comment by ojhall94** [comment]
Hi Geert,

Your reasoning is sound, and I agree; my experience seems to stem from me not using pandas in the way it was intended, so I learned something new today! I will close the issue.
